zyte_api/apikey.py

```diff
@@ -14,7 +14,7 @@ def get_apikey(key: Optional[str] = None) -> str:
     if key is not None:
```

Comment:
Our **mypy** checks were recently failing because of the change below. This function should be updated into:

```
def get_apikey(key: Optional[str] = None) -> Optional[str]:
```

Reply:
I have reverted the change that triggered this error.